src/components/Navigation/README.md

```diff
@@ -127,6 +127,7 @@ The content of the navigation component consists of navigation items. Each item
 | badge              | string \| null      | A string property allowing content to be displayed in a badge next to the navigation item                         |
 | label              | string              | A string property allowing content to be displayed as link text in the navigation item                            |
 | disabled           | boolean             | A boolean property indicating whether the navigation item is disabled                                             |
+| new                | boolean             | A boolean property indicating whether the navigation item is new                                                  |
```

Comment:
Should we mention that this adds the indicator dot to the parent? And that it will be overridden by the `badge` prop?

Reply:
(I don’t think the descriptions here need to give the datatype either; there’s a separate column for that)